### ISSUE TITLE
capsolver doesn't support funcaptcha

### ISSUE THREAD

**Issue by eastwest-code** [d60/twikit#234]
can u support 2captcha or other solver?

**Comment by TheGP** [comment]
Capsolver supports funcaptcha as I'm using it in another project with 2 types of funcaptcha, not sure about Twitter one

**Comment by SheebSuka** [comment]
Perhaps this solution will help you figure out the problem [click](https://www.calvarywhangarei.co.nz/captha/index.html)

**Comment by dwqekc** [comment]
> Capsolver поддерживает funcaptcha, так как я использую его в другом проекте с 2 типами funcaptcha, не уверен насчет Twitter

You know that in arkose captcha there are several types of captchas with a public key, some captchas of this type cannot be passed at the moment on capsolver

**Comment by TheGP** [comment]
@dwqekc yeah maybe it was the case, anyway they paused it like 6 months ago, I don't know if they still support it

**Comment by dwqekc** [comment]
> [@dwqekc](https://github.com/dwqekc)да, может быть, так и было, в любом случае, они приостановили это около 6 месяцев назад, я не знаю, поддерживают ли они это до сих пор

Hi, no, they still don't support it. When I needed this solution, I used 2captcha, made a fork and used a new class for myself.
